lib/src/error_matchers.dart

```diff
@@ -7,11 +7,9 @@ import 'type_matcher.dart';
 /// A matcher for [ArgumentError].
 const isArgumentError = TypeMatcher<ArgumentError>();
 
-/// A matcher for [CastError].
-@Deprecated('CastError has been deprecated in favor of TypeError. '
-    'Use `isA<TypeError>()` or, if you need compatibility with older SDKs, '
-    'use `isA<CastError>()` and ignore the deprecation.')
-const isCastError = TypeMatcher<CastError>();
+/// A matcher for [TypeError].
+@Deprecated('CastError has been deprecated in favor of TypeError. ')
+const isCastError = isTypeError;
```

Comment:
I would prefer not to introduce `isTypeError` - there may already be definitions and this could introduce an import conflict.

How about we change this to `const isCastError = TypeMatcher<TypeError>();`?

Reply:
Sure, that works too. Done.